test/unit/lib/plugins/aws/package/compile/events/stream.test.js

```diff
@@ -1,11 +1,14 @@
 'use strict';
 
-const expect = require('chai').expect;
+const chai = require('chai');
 const AwsProvider = require('../../../../../../../../lib/plugins/aws/provider');
 const AwsCompileStreamEvents = require('../../../../../../../../lib/plugins/aws/package/compile/events/stream');
 const Serverless = require('../../../../../../../../lib/serverless');
 const runServerless = require('../../../../../../../utils/run-serverless');
 
+const { expect } = chai;
+chai.use(require('chai-as-promised'));
+
 describe('AwsCompileStreamEvents', () => {
```

Comment:
Let's not modify or restructure old tests.

They either should be removed or kept intact. Otherwise let's not imply any work on them